ogre2/src/Ogre2GpuRays.cc

```diff
@@ -439,6 +620,7 @@ void Ogre2GpuRays::Setup1stPass()
   // {
   //   in 0 rt_input
   //   texture depthTexture target_width target_height PF_D32_FLOAT
+  //   texture colorTexture target_width target_height PF_R8G8B8
   //   target depthTexture
```

Comment:
since we removed the depthTexture target pass in this PR, you can remove this part of the comment.

Reply:
Did you mean this line ? :
https://github.com/ignitionrobotics/ign-rendering/blob/6f27238678eb71567d698c68430a5ec8f56db9f7/ogre2/src/Ogre2GpuRays.cc#L622

And I guess this line too ?:
https://github.com/ignitionrobotics/ign-rendering/blob/6f27238678eb71567d698c68430a5ec8f56db9f7/ogre2/src/Ogre2GpuRays.cc#L653

and modifying https://github.com/ignitionrobotics/ign-rendering/blob/6f27238678eb71567d698c68430a5ec8f56db9f7/ogre2/src/Ogre2GpuRays.cc#L654 to `input 0 colorTexture`

(I am not familiar with the shader syntax)

Reply:
we're still creating the depth texture and feeding that to the `rt_input` target pass. But we are no longer rendering the `depthTexture` target pass. So we can remove these lines: https://github.com/ignitionrobotics/ign-rendering/blob/6f27238678eb71567d698c68430a5ec8f56db9f7/ogre2/src/Ogre2GpuRays.cc#L624-L633